ui/design-system-core/src/components/Icon/index.tsx

```diff
@@ -24,18 +21,9 @@ export interface IconProps {
   solid?: boolean;
 }
 
-const fillOnlyIcons: IconType[] = [
-  'akasha',
-  'discord',
-  'telegram',
-  'twitter',
-  'widget',
-  'metamask',
-];
-
```

Comment:
Here is what the fill only icons look like on your PR :

![image](https://github.com/AKASHAorg/akasha-core/assets/35398733/4269b12c-aa83-4026-b94a-6bae3416db51)

Can your revisit how you are setting the solid prop for those and if the style is being applied correctly ?

Reply:
Alright, let me take a look